registry/tracecat_registry/templates/checkpoint/get_cp_xdr_incidents.yml

```
@@ -0,0 +1,23 @@
+type: action
+definition:
+  name: get_cp_xdr_incidents
+  namespace: integrations.checkpoint
+  title: Get Check Point Infinity XDR incidents for the last 7 days
+  description: Retrieve Check Point Infinity XDR incidents for the last 7 days.
+  display_group: Check Point
+  expects:
+    filterBy:
+      type: str
+      description: Filter by createdAt or updatedAt
+      default: updatedAt
+  steps:
+    - ref: get_cp_xdr_incidents
```

```suggestion
    - ref: get_auth_token
       action: integrations.checkpoint.get_auth_token
 
    - ref: get_cp_xdr_incidents
```

Let's abstract the JWT token call into the action template itself!

---

My bad. This need more testing. I am getting this error:
`2024-11-10 17:45:49 api                   | 2024-11-10 15:45:49.343544Z [10] | ERROR     tracecat.registry.repository:load_template_actions_from_path:462 - Could not parse /app/registry/tracecat_registry/templates/checkpoint/get_xdr_incidents.yml as template action, skipped | {'error': 1 validation error for TemplateAction
2024-11-10 17:45:49 api                   | definition.steps.0.args
2024-11-10 17:45:49 api                   |   Field required [type=missing, input_value={'ref': 'get_auth_token',...ckpoint.get_auth_token'}, input_type=dict]
2024-11-10 17:45:49 api                   |     For further information visit https://errors.pydantic.dev/2.6/v/missing}`